Dropped support for Python 3.8 and Django 5.0

clintonb opened the pull request:
Both Python 3.8 and Django 5.0 have reached EOL, so are no longer supported by this package.

clintonb committed:
Dropped support for Python 3.8 and Django 5.0

Both Python 3.8 and Django 5.0 have reached EOL, so are no longer supported by this package.

dancergraham commented on pyproject.toml:
```diff
@@ -68,7 +66,7 @@ ignore_missing_imports = true
 [tool.ruff]
 line-length = 120
-target-version = "py38"
+target-version = "py313"
```
should be py39 to avoid ruff suggesting incompatible upgrades

dancergraham commented on pyproject.toml:
```diff
@@ -55,7 +53,7 @@ namespaces = false
 
 [tool.mypy]
-python_version = "3.8"
+python_version = "3.13"
```
should be 3.9 to avoid mypy failing on older syntax

dancergraham committed:
fix: linting on the lowest supported version

Type checking and linting should be on the lowest supported version to avoid breaking things for users
